examples/xml/pincell_multigroup/mg_cross_sections.xml

```diff
@@ -1,383 +0,0 @@
-    </total>
-  </xsdata>
-
-  <xsdata>
-    <!-- Meta data for this data -->
-    <name>LWTR</name>
-    <alias>LWTR</alias>
-    <kT> 2.53E-8 </kT> <!-- in MeV -->
-    <order>0</order>
-    <fissionable>false</fissionable>
-
-    <!-- The data itself, like tallies,
-      goes from low energies (groups) to high energies
-    -->
-    <absorption>
-      6.0105E-04 1.5793E-05 3.3716E-04 1.9406E-03 5.7416E-03 1.5001E-02 3.7239E-02
-    </absorption>
-
-    <!-- for consistency must include nu-scatter -->
-    <!-- will be a matrix of (order+1) x g_in x g_out -->
-    <scatter>
-      0.0444777 0.1134000 0.0007235 0.0000037 0.0000001 0.0000000 0.0000000
-      0.0000000 0.2823340 0.1299400 0.0006234 0.0000480 0.0000074 0.0000010
-      0.0000000 0.0000000 0.3452560 0.2245700 0.0169990 0.0026443 0.0005034
-      0.0000000 0.0000000 0.0000000 0.0910284 0.4155100 0.0637320 0.0121390
-      0.0000000 0.0000000 0.0000000 0.0000714 0.1391380 0.5118200 0.0612290
-      0.0000000 0.0000000 0.0000000 0.0000000 0.0022157 0.6999130 0.5373200
-      0.0000000 0.0000000 0.0000000 0.0000000 0.0000000 0.1324400 2.4807000
-    </scatter>
-
-    <total>
-      0.15920605  0.41296959299999997 0.59030986  0.5843499999999999  0.7180000000000001  1.2544497000000001  2.650379
-    </total>
-
-  </xsdata>
-
-  <xsdata>
-    <!-- Meta data for this data -->
-    <name>CR</name>
-    <alias>CR</alias>
-    <kT> 2.53E-8 </kT> <!-- in MeV -->
-    <order>0</order>
-    <fissionable>false</fissionable>
-
-    <!-- The data itself, like tallies,
-      goes from low energies (groups) to high energies
-    -->
-    <absorption>
-      1.70490000E-03 8.36224000E-03 8.37901000E-02 3.97797000E-01 6.98763000E-01 9.29508000E-01 1.17836000E+00
-    </absorption>
-
-    <!-- for consistency must include nu-scatter -->
-    <!-- will be a matrix of (order+1) x g_in x g_out -->
-    <scatter>
-      1.70563000E-01  4.44012000E-02  9.83670000E-05  1.27786000E-07  0.00000000E+00  0.00000000E+00  0.00000000E+00
-      0.00000000E+00  4.71050000E-01  6.85480000E-04  3.91395000E-10  0.00000000E+00  0.00000000E+00  0.00000000E+00
-      0.00000000E+00  0.00000000E+00  8.01859000E-01  7.20132000E-04  0.00000000E+00  0.00000000E+00  0.00000000E+00
-      0.00000000E+00  0.00000000E+00  0.00000000E+00  5.70752000E-01  1.46015000E-03  0.00000000E+00  0.00000000E+00
-      0.00000000E+00  0.00000000E+00  0.00000000E+00  6.55562000E-05  2.07838000E-01  3.81486000E-03  3.69760000E-09
-      0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  1.02427000E-03  2.02465000E-01  4.75290000E-03
-      0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  3.53043000E-03  6.58597000E-01
-    </scatter>
-
-    <total>
-      2.16767595E-01 4.80097720E-01 8.86369232E-01 9.70009150E-01 9.10481420E-01 1.13775017E+00 1.84048743E+00
-    </total>
-  </xsdata>
-</library>
```

Comment:
Does the "mgxs.h5" file need to be shipped with the code? It looks like it is created each time the code is run, and hence does not need to be version controlled (nor should it be)